datastore/data-loader/client_knmi_station.py

```diff
@@ -61,6 +78,32 @@ def netcdf_file_to_requests(file_path: Path | str) -> Tuple[List, List]:
     return observation_request_messages
 
 
+def generate_parameter_name(standard_name, level, long_name, instrument):
+    if "Minimum" in long_name:
+        function = "minimum"
+    elif "Maximum" in long_name:
+        function = "maximum"
+    elif "Average" in long_name:
+        function = "mean"
+    else:
+        function = "point"
+
+    if period_raw := re.findall(r"(\d+) (Hours|Min)", long_name):
+        if len(period_raw) == 1:
+            period_raw = period_raw[0]
+        else:
+            raise Exception(f"{period_raw}, {long_name}")
+        time, scale = period_raw
+        if scale == "Hours":
+            period = f"PT{time}H"
+        elif scale == "Min":
+            period = f"PT{time}M"
+    else:
+        period = "PT0S"
```

Comment:
Although period is scoped inside the function. I think it is easier to read when placed above the if statement as default for the return statement below.

```suggestion

    period = "PT0S"
    if period_raw := re.findall(r"(\d+) (Hours|Min)", long_name):
        if len(period_raw) == 1:
            period_raw = period_raw[0]
        else:
            raise Exception(f"{period_raw}, {long_name}")
        time, scale = period_raw
        if scale == "Hours":
            period = f"PT{time}H"
        elif scale == "Min":
            period = f"PT{time}M"
```